src/main/java/se/diabol/jenkins/pipeline/domain/Pipeline.java

```diff
@@ -233,7 +243,7 @@ public Pipeline createPipelineAggregated(ItemGroup context, boolean showAggregat
      *
      * @param noOfPipelines number of pipeline instances
      */
-    public List<Pipeline> createPipelineLatest(int noOfPipelines, ItemGroup context, boolean pagingEnabled) {
+    public List<Pipeline> createPipelineLatest(int noOfPipelines, ItemGroup context, boolean pagingEnabled, Component component) {
         List<Pipeline> result = new ArrayList<Pipeline>();
```

Comment:
According to configured in ComponentSpecs, each ComponentSpec created as Component and pipelines are associated to this Component. We are passing each respective Component while getting pipelines.